challenge/src/App.jsx

```diff
@@ -11,6 +11,10 @@ class App extends Component {
     }
     render() {
         return (
+            console.log("I am rendering successfully!"),
```

Comment:
returning the console statement seems a little wonky, since console.log doesn't have a result. is there some where's else we can return that?